package.json

```diff
@@ -1,29 +1,40 @@
 {
   "name": "scribe-editor",
-  "version": "3.3.0",
+  "version": "4.0.0",
   "license": "Apache-2.0",
   "main": "src/scribe.js",
   "dependencies": {
-    "immutable": "~3.8.1"
+    "immutable": "^3.8.1"
   },
   "devDependencies": {
-    "chai": "~1.9.1",
-    "grunt": "^1.0",
-    "grunt-contrib-requirejs": "^1.0.0",
-    "grunt-mocha-test": "^0.12.7",
-    "grunt-cli": "^0.1",
-    "http-server": "~0.6.1",
-    "mocha": "~2.4",
-    "mock-browser": "0.92.11",
-    "mversion": "~1.10",
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.24.1",
+    "babel-preset-es2015": "^6.24.1",
+    "chai": "~4.1.1",
+    "html-janitor": "^2.0.2",
+    "http-server": "~0.10.0",
+    "mocha": "~3.5.0",
+    "mock-browser": "0.92.14",
+    "mversion": "^1.10.1",
     "node-amd-require": "^0.2.2",
-    "q": "~1.2",
-    "request": "~2.33.0",
-    "sinon": "^1.12.2"
+    "q": "~1.5.0",
+    "request": "~2.81.0",
+    "requirejs": "~2.1.9",
+    "scribe-plugin-blockquote-command": "guardian/scribe-plugin-blockquote-command#master",
```

Comment:
Are you missing https://www.npmjs.com/package/scribe-common from the package.json?